### ISSUE TITLE
Persistence of enum choices in Commands

### ISSUE THREAD

**Issue by stefanhahmann** [scijava/scijava-common#471]
With https://github.com/scijava/scijava-common/commit/664b3ab482f7cda3303154507b7e515a48cc6dee it was made possible to use enums with custom labels in scijava commands

However, currently enums with custom demands are not persisted, i.e. if the user makes some settings using an enum with custom labels in a dialog, closes this dialog and comes back to this dialog, the selected value would fall back to the default parameter instead of the last selected parameter.

**Comment by imagejan** [comment]
I noticed that persisted enum values are actually harvested just fine: as long as you _don't touch the dropdown widget_, the persisted value is kept. It's just the UI that isn't updated to reflect the persisted value 🤦 

Consider this Groovy script:
```
#@ org.scijava.ItemIO itemio

println itemio
```

When you run it, the dialog will always show the first enum value, `INPUT`:

![Image](https://github.com/user-attachments/assets/55b90188-01c0-4d8e-a5ec-6a619978c26a)

However, if you choose an alternative value (e.g. `BOTH`), press OK, and re-run the script, this time without changing in the dropdown (that again shows `INPUT`), the correct, persisted value (i.e. `BOTH`) will be printed.

That means this issue actually belongs to `scijava-ui-swing` 🚀 .

**Comment by tischi** [comment]
I am sorry 😬 ! 

I did not carefully read @imagejan's above post. 
In fact it seems that he exactly found what I found.
I also saw now that the PR has not been merged yet.

**tl;dr**   please simply ignore all my comments.
